### DISCUSSION TITLE
Set default value for Options Picker

### DISCUSSION THREAD

**Discussion by burnoutberni** [Budibase/budibase#1903] [Ideas]
Is there a way to set a default value for an Options Picker in a form? I'd also be happy, if I could set a general default value for the referenced table column. Unfortunately, I can't make it work at the moment.

**Comment by joebudi** [comment]
I don't believe there is a way (for both) sadly, but it is a common use case. I am going to change the category of this discussion to Ideas so we can catch it when planning our next iteration.

Thanks for raising this.

**Reply by NicolasFabre81** [reply]
Is there a workaround or solution now ?

**Reply by kasgithub8** [reply]
@NicolasFabre81 yea, there's a "default value" for the option picker.

**Comment by AkinoYuki** [comment]
Couldn't find in the documentation how we can set the default value of an multi select picker. I have multiple values in the table, and would like to initialize the multiselect or option picker with that/those value(s). Can you document that please?

**Comment by NicolasFabre81** [comment]
Just put the list of values with commas : a,b,c
